### ISSUE TITLE
When using SMW query with format=gantt, a colon in the "task" value for a page in the results will prevent the chart from being generated

### ISSUE THREAD

**Issue by darenwelsh** [SemanticMediaWiki/Mermaid#65]
### Setup and configuration

- Mermaid version: 2.1.1
- MediaWiki version: 1.31.12 (ef13e11)
- PHP version: 7.2.34 (apache2handler)
- Database system (MySQL, PostgresQL, etc.) and version: 5.5.68-MariaDB

### Issue

I noticed today that a Gantt chart was not rendering. It had worked for months. I finally isolated the culprit to a wiki page that had a colon ":" as part of the value for the SMW property "Title" (not PAGENAME, just a property value). This page did not have the colon character for a long time and then just recently someone decided to change it that way. When I changed this value to not include the colon character, everything worked fine again.

**Comment by kghbln** [comment]
Thanks for reporting. @gesinn-it FYI: I believe you are intensely using this format.

**Comment by gesinn-it** [comment]
@kghbln no, we are mainly using Mermaid as part of SRF/Gantt https://github.com/SemanticMediaWiki/SemanticResultFormats/tree/master/formats/Gantt. @darenwelsh can you try to find out, if SRF/Gantt is also affected by this issue?

**Comment by darenwelsh** [comment]
I don't understand how these are different. This seems exactly like what I'm doing. https://www.semantic-mediawiki.org/wiki/Help:Gantt_format

**Comment by gesinn-it** [comment]
https://github.com/mermaid-js/mermaid/issues/742
